### PR TITLE
[contracts] Interpreter renaming

### PR DESCRIPTION
**Pull request by cf19drofxots**
This adds more precise names for interpreters.

### CODE REVIEW

File: packages/contracts/contracts/interfaces/Interpreter.sol [hunk 1]
```diff
@@ -6,7 +6,8 @@ contract Interpreter {
 
   enum OutcomeType {
     TWO_PARTY_OUTCOME,
-    ETH_TRANSFER
+    ETH_TRANSFER,
+    TOKEN_TRANSFER
```
Comment (snario) [inline packages/contracts/contracts/interfaces/Interpreter.sol]: Would name it to `TWO_PARTY_NUMERIC_OUTCOME`

File: packages/contracts/contracts/interpreters/ERC20Interpreter.sol [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+
+import "../interfaces/Interpreter.sol";
+
+
+/**
+ * This file is excluded from ethlint/solium because of this issue:
+ * https://github.com/duaraghav8/Ethlint/issues/261
+ */
+contract ERC20Interpreter is Interpreter {
```
Comment (snario) [inline packages/contracts/contracts/interpreters/ERC20Interpreter.sol]: Maybe `ERC20TwoPartyNumericInterpreter`

**Comment by cf19drofxots** [comment]
Currently on pause for a short period until work on productionizing postgres has been done.

File: packages/apps/contracts/EthUnidirectionalPaymentApp.sol [hunk 3]
```diff
@@ -2,7 +2,8 @@ pragma solidity 0.5.9;
 pragma experimental "ABIEncoderV2";
 
 import "@counterfactual/contracts/contracts/interfaces/CounterfactualApp.sol";
-import "@counterfactual/contracts/contracts/interfaces/TwoPartyOutcome.sol";
+// solium-disable-next-line
```
Comment (IIIIllllIIIIllllIIIIllllIIIIllllIIIIll) [inline packages/apps/contracts/EthUnidirectionalPaymentApp.sol]: can this be in a separate PR?

Comment (cf19drofxots) [reply packages/apps/contracts/EthUnidirectionalPaymentApp.sol]: Actually most of this PR is not even related to the work needed for the title of the PR, so I can remove the minimal ERC20 stuff from this PR and treat this as a naming update PR.

Comment (cf19drofxots) [reply packages/apps/contracts/EthUnidirectionalPaymentApp.sol]: Updated PR to reflect above comment.
